test/fuzz/jbig2.fuzz.js

```diff
@@ -0,0 +1,22 @@
+import { Jbig2Image } from "../../build/image_decoders/pdf.image_decoders.mjs";
+
+const ignored = ["Cannot read properties", "JBIG2 error"];
+
+function ignoredError(error) {
+  return ignored.some(message => error.message.includes(message));
+}
+
+/**
+ * @param {Buffer} data
+ */
```

Comment:
Sorry, but this isn't correct; this comment applies in the files below as well.

Please note that PDF.js library does *not* support Node.js `Buffer`s, note [this code](https://github.com/mozilla/pdf.js/blob/f4b396f6c88e951174879a52390fa89223326b36/src/display/api.js#L534-L544), which thus (implicitly) applies to the image decoders as well.
```suggestion
/**
 * @param {Uint8Array} data
 */
```